src/evo/evodb.cpp

```diff
@@ -32,14 +32,16 @@ void CEvoDBScopedCommitter::Rollback()
     evoDB.RollbackCurTransaction();
 }
 
-CEvoDB::CEvoDB(size_t nCacheSize, bool fMemory, bool fWipe) :
-    db(fMemory ? "" : (gArgs.GetDataDirNet() / "evodb"), nCacheSize, fMemory, fWipe),
-    rootBatch(db),
-    rootDBTransaction(db, rootBatch),
-    curDBTransaction(rootDBTransaction, rootDBTransaction)
+CEvoDB::CEvoDB(const util::DbWrapperParams& db_params) :
+    db{util::MakeDbWrapper({db_params.path / "evodb", /*cache_size=*/64 << 20, db_params.memory, db_params.wipe})},
```

Comment:
if dropped gArgs, should drop system.h also I think, is there any other dependency? I can't find any

```
-#include <util/system.h>
```